samples/pmic/native/npm13xx_one_button/README.rst

```diff
@@ -16,24 +16,24 @@ The sample supports the following development kits:
 
 .. table-from-sample-yaml::
 
-The sample also requires an `nPM1300 EK <nPM1300 EK product page_>`_ that you need to connect to the development kit as described in `Wiring`_.
+The sample also requires an `nPM1300 EK <nPM1300 EK product page_>`_ or an nPM1304 EK that you need to connect to the development kit as described in `Wiring`_.
```

Comment:
Link to product page missing.

Reply:
You've added it to links.txt in this PR. links.txt is not parsed by CI like :ref:s
```suggestion
The sample also requires an `nPM1300 EK <nPM1300 EK product page_>`_ or an `nPM1304 EK <nPM1304 EK product page_>`_ that you need to connect to the development kit as described in `Wiring`_.
```